src/controllers/userController.js

```diff
@@ -0,0 +1,69 @@
+const {
+  getUsers,
+  getUserById,
+  create,
+  remove,
+  update,
+} = require('../services/userService');
+
+const getAll = (req, res) => {
+  res.statusCode = 200;
+  res.send(getUsers());
+};
+
+const getUser = (req, res) => {
+  const { id } = req.params;
+
+  const user = getUserById(+id);
```

Comment:
The `getUserById` function is called synchronously. If this function involves database or I/O operations, it should be made asynchronous to prevent blocking the event loop.